### PR TITLE
WIP: Add multisite support for admin page

### PR DESCRIPTION
**Pull request by vishalkakadiya**
Adds multisite support for admin page

- Addresses the acceptance criteria in https://github.com/10up/plugin-check/issues/64

Closes https://github.com/10up/plugin-check/issues/64

### CODE REVIEW

File: includes/Admin/Admin_Page.php [hunk 1]
```diff
@@ -49,13 +57,27 @@ public function add_hooks() {
...
-			array( $this, 'render_page' )
-		);
+
+		if ( is_multisite() ) {
+
+			$hook = add_submenu_page(
+				'settings.php',
+				__( 'Plugin Check', 'plugin-check' ),
+				__( 'Plugin Check', 'plugin-check' ),
+				'activate_plugins',
```
Comment (spacedmonkey) [inline includes/Admin/Admin_Page.php:67]:
```suggestion
				'manage_network_plugins',
```

[manage_network_plugins](https://wordpress.org/documentation/article/roles-and-capabilities/#manage_network_plugins)

File: includes/Admin/Admin_Page.php [hunk 2]
```diff
@@ -49,13 +57,27 @@ public function add_hooks() {
...
+				'plugin-check',
+				array( $this, 'render_page' )
+			);
+		} else {
+
+			$hook = add_management_page(
+				__( 'Plugin Check', 'plugin-check' ),
+				__( 'Plugin Check', 'plugin-check' ),
+				'activate_plugins',
```
Comment (spacedmonkey) [inline includes/Admin/Admin_Page.php:76]:
```suggestion
				'manage_network_plugins',
```

**Comment by spacedmonkey** [comment]
I would look into this line as well 
https://github.com/10up/plugin-check/blob/08df399884b13ecefde47a896892f6a29f200845/includes/Admin/Admin_Page.php#L147 
and maybe also check for can activate network plugins.

**Comment by spacedmonkey** [comment]
Screenshot 2023-03-17 at 11 48 06

Should this button be styled?

**Comment by spacedmonkey** [comment]
I would consider adding a property to the class to stop repeated code. See https://github.com/spacedmonkey/plugin-check/commit/f4d2244b27ffbd864c98848f9c50d239a7b3a71a

**Comment by swissspidy** [comment]
Closing this one for now as it's stale and has conflicts. It can be reopened / forked when the issue gets picked up again.

**Pull request closed without merging**